dali/util/std_cufile.cc

```diff
@@ -106,7 +106,8 @@ void StdCUFileStream::HandleIOError(int64 ret) const {
   if (ret == -1) {
     std::string errmsg(256, '\0');
     int e = errno;
-    strerror_r(e, &errmsg[0], errmsg.size());
+    auto ret = strerror_r(e, &errmsg[0], errmsg.size());
+    DALI_ENFORCE(ret == 0, make_string("Unknown CUFile error: ", e));
```

Comment:
Nitpick: I think that this condition isn't very informative for the reader.
```suggestion
    if (ret != 0) {
      DALI_FAIL(make_string("Unknown CUFile error: ", e));
    }
```

Reply:
Done